src/client/Microsoft.Identity.Client/OAuth2/OAuth2Client.cs

```diff
@@ -146,8 +151,9 @@ private bool AddCommonHeaders(RequestContext requestContext, bool addCorrelation
                 _headers.Add(OAuth2Header.AppVer, requestContext.Logger.ClientVersion);
             }
 
-            _headers.Add(TelemetryConstants.XClientLastTelemetry, _telemetryManager.FetchAndResetPreviousHttpTelemetryContent());
-            _headers.Add(TelemetryConstants.XClientCurrentTelemetry, _telemetryManager.FetchCurrentHttpTelemetryContent(requestContext.CorrelationId.AsMatsCorrelationId()));
+            //_headers.Add(TelemetryConstants.XClientLastTelemetry, _telemetryManager.FetchAndResetPreviousHttpTelemetryContent());
+            //_headers.Add(TelemetryConstants.XClientCurrentTelemetry, _telemetryManager.FetchCurrentHttpTelemetryContent());
+
```

Comment:
Is the above code left commented for some reason?

Reply:
This is work in progress, Jenny asked for an initial review to see if approach is valid. Apologies for not making this clear when I created a PR off her branch.